docs/dev/table/connectors/formats/canal.zh.md

```diff
@@ -189,6 +189,12 @@ metadata fields for its value format.
       <td>The timestamp at which the connector processed the event. Corresponds to the <code>ts</code>
       field in the Canal record.</td>
     </tr>
+    <tr>
+      <td><code>event-timestamp</code></td>
+      <td><code>TIMESTAMP(3) WITH LOCAL TIME ZONE NULL</code></td>
+      <td>The timestamp at which the connector produced the event. Corresponds to the <code>es</code>
```

Comment:
This description is misleading, because "the connector produced the event" sounds "ingetsion time". The event and the `es` is not produced by the connector, but the binlog dump of mysql server. Maybe we can say:

```
The timestamp when the corresponding change was executed in MySQL server.
```